content/tutorials/ably-kafka-connector.textile

```diff
@@ -0,0 +1,115 @@
+Once the data has been stored and processed, we need a way to make the result available to users of our service, so they can see where our drivers are.
+In order to allow for all these users to access the data, we need another service to handle this distribution of data. Ably is the perfect candidate for this, as not only can it provide low-latency, highly reliable data distribution to millions of users, but it provides many of the additional bells and whistles you’d expect to keep your service secure and functional, such as "Token Authentication":/core-features/authentication.
+
+The Ably Kafka Connector - a sink connector built on top of Kafka Connect - makes it simple to map your Kafka topics to Ably channels, and handles all the complexity involved in getting the data out of Kafka and into Ably.
+
+<%= partial partial_version('tutorials/_step-1-create-app-and-api-key') %>
+
+h2(#setting-up-kafka-connector). Setting up the Ably Kafka Connector
+
+The Ably Kafka Connector is open-source and available on "GitHub":https://github.com/ably/kafka-connect-ably, as well as "on Confluent as a Verified Gold Connector":https://www.confluent.io/hub/ably/kafka-connect-ably/. In this tutorial we'll be building it into a Docker Image, so you'll need to "install Docker":https://docs.docker.com/get-docker/ first. Once you've installed it, check it's working correctly by typing @docker help@ into the terminal. If it is installed correctly, you should get a bunch of information on the usage of docker as a response.
+
+Once you have Docker, we can "download the Ably Kafka Connector from GitHub":https://github.com/ably/kafka-connect-ably. This repository contains all of the crucial code required to create the Ably Kafka Connector. There are some notable files in here which you should be aware of:
+
+* docker-compose.yml - This Docker Compose file defines not only the building of a Docker Image for the Ably Kafka Connector, but also sets up Zookeeper, Kafka, and the necessary network settings to allow for all 3 of these images to eventually interact with one another once constructed into Containers
+* config/example-connector.properties - This contains a sample of the config settings you'll need for the Connector to work. Notably within it, @topics@ defines the Kafka Topics you want the Connector to apply to, and @channel@ defines the Ably Channel you'll want to publish in to.
+
+For this tutorial, we'll be largely keeping our config details the same as those within @example-connector.properties@. The Connector is expecting a config file to exist as @config/docker-compose-connector.properties@, so let's copy the example file to that:
+
+```[sh]
+cp config/example-connector.properties config/docker-compose-connector.properties
+```
+
+The contents of this file should roughly be:
+
+```[txt]
+name = ably-channel-sink
+topics = kafka-connect-ably-example
+tasks.max = 1
+connector.class = com.ably.kafka.connect.ChannelSinkConnector
```

Comment:
This is currently required but honestly I think this should already be embedded as default connector. I will see if I can make it optional and then we might remove this from the instructions.

Reply:
This one doesn't look possible so I think this can be left as it is